docs/redirects.map

```diff
@@ -10,7 +10,7 @@ daml/anti-patterns.html
 daml/models-versus-application.html
 daml/stdlib/base.html -> daml/reference/base.html
 concepts.html -> /concepts/ledger-model/index.html
-getting-started.html -> /getting-started/quickstart.html
+getting-started.html -> /app-dev/bindings-java/quickstart.html
```

Comment:
I think we should point this to the new GSG. We are allowed to change the content of pages, we just don't want dead links.

Reply:
I don't actually know what this redirects map is, but I made the change.